code/code/disc/disc_adv_adventuring.cc

```diff
@@ -834,8 +834,6 @@ void TBeing::doSeekwater()
   }
 
   // Let's determine the distance:
-  // Ranges: 21, ..., 120
-  dist = (max(5, lev)/5) * (max(10, skill)/10) + 20;
   dist = lev * max(10, skill);
```

Comment:
I dunno, if you think we should investigate we should do it on a separate ticket. I just removed a line of dead code. Should have left the comment in though, I'll add it back, along with the removed line as a comment.